ci-scripts/JenkinsFile-GitLab

```diff
@@ -310,32 +311,32 @@ pipeline {
 
                 echo "Starting the integration Tests - S1AP Tester"
                 // We have removed the traffic testcases from mandatory suite.
-                timeout (time: 20, unit: 'MINUTES') {
+                timeout (time: 45, unit: 'MINUTES') {
                   myShCmdWithLog('cd lte/gateway && vagrant ssh magma_test -c "cd magma/lte/gateway/python/integ_tests/ && source ~/build/python/bin/activate && make integ_test"', 'archives/magma_run_s1ap_tester.log', new_host_flag, new_host_user, new_host)
                 }
                 // Adding a few tests not in the mandatory suite and the UL traffic scenarios (launched manually)
-                timeout (time: 45, unit: 'SECONDS') {
+                timeout (time: 45, unit: 'MINUTES') {
                   try {
                     myShCmdWithLogAppend('cd lte/gateway && vagrant ssh magma_test -c "cd magma/lte/gateway/python/integ_tests/ && source ~/build/python/bin/activate && make integ_test TESTS=s1aptests/test_attach_ul_udp_data.py"', 'archives/magma_run_s1ap_tester.log', new_host_flag, new_host_user, new_host)
                   } catch (Exception e) {
                     echo "s1aptests/test_attach_ul_udp_data testcase may fail"
                   }
                 }
-                timeout (time: 45, unit: 'SECONDS') {
+                timeout (time: 45, unit: 'MINUTES') {
                   try {
                     myShCmdWithLogAppend('cd lte/gateway && vagrant ssh magma_test -c "cd magma/lte/gateway/python/integ_tests/ && source ~/build/python/bin/activate && make integ_test TESTS=s1aptests/test_attach_ul_tcp_data.py"', 'archives/magma_run_s1ap_tester.log', new_host_flag, new_host_user, new_host)
                   } catch (Exception e) {
                     echo "s1aptests/test_attach_ul_tcp_data testcase may fail"
                   }
                 }
-                timeout (time: 45, unit: 'SECONDS') {
+                timeout (time: 45, unit: 'MINUTES') {
```

Comment:
This is a single test run, so timing out in 45 seconds is the correct way to go.